README.md

```diff
@@ -235,7 +235,7 @@ You can get the current version of the sda-cli by running:
 This section contains the information required to install, modify and run the `sda-cli` tool.
 
 ## Requirements
-The `sda-cli` is written in golang. In order to be able to modify, build and run the tool, golang (>= 1.20) needs to be installed. The instructions for installing `go` can be found [here](https://go.dev/doc/install).
+The `sda-cli` is written in golang. In order to be able to modify, build and run the tool, golang (>= 1.21) needs to be installed. The instructions for installing `go` can be found [here](https://go.dev/doc/install).
```

Comment:
```suggestion
The `sda-cli` is written in golang. In order to be able to modify and build the tool, golang (>= 1.21) needs to be installed. The instructions for installing `go` can be found [here](https://go.dev/doc/install).

Binaries built do not require the golang environment and can be moved to other machines with the same OS and processor architecture and run there.
```

Also clarify what can be interpreted as all users needing to have golang installed?

Reply:
This part is under the `Developer's section`, so this applies to developers not all users. Not sure what needs to be clarified further :-)